x/ccv/consumer/types/params.go

```diff
@@ -1,15 +1,28 @@
 package types
 
 import (
-	"time"
+	fmt "fmt"
+	time "time"
 
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 	ccvtypes "github.com/cosmos/interchain-security/x/ccv/types"
 )
 
 const (
 	// about 2 hr at 7.6 seconds per blocks
 	DefaultBlocksPerDistributionTransmission = 1000
+
+	// 1 week
+	DefaultTransferTimeoutPeriod = 1 * 7 * 24 * time.Hour
+
+	// The default fraction of tokens allocated to the consumer redistribution address
+	// during distribution events. The fraction is a string representing a
+	// decimal number. For example "0.75" would represent 75%.
+	DefaultConsumerRedistributeFrac = "0.75"
+
+	// Default number of historical info entries to persist in store
+	DefaultNumHistoricalEntries = 10000
```

Comment:
Could we use this as default: https://github.com/cosmos/cosmos-sdk/blob/df4d6d1a4cd9fa0247f9db9378db857d95a1c1cb/x/staking/types/params.go#L32